### ISSUE TITLE
Commands to reproduce the ViT-T and ViT-S experiments

### ISSUE THREAD

**Issue by LiuDongyang6** [OliverRensu/TinyMIM#5]
Thanks for this work!

Would you like to share the commands of the experiments with ViT-T and ViT-S as the student? I am interested in the MAE-VIT-B -> VIT-S and the MAE-VIT-B -> VIT-T experiments in Table 11, and I would like to know if any hyperparameters need to be changed.

**Comment by OliverRensu** [comment]
The pretraining commands are identical. During finetuning, we adpot a higher learning rate and larger batch size.
